addons/hr_expense/models/hr_expense.py

```diff
@@ -72,7 +72,8 @@ def _get_employee_id_domain(self):
     quantity = fields.Float(required=True, readonly=True, states={'draft': [('readonly', False)], 'reported': [('readonly', False)], 'refused': [('readonly', False)]}, digits='Product Unit of Measure', default=1)
     tax_ids = fields.Many2many('account.tax', 'expense_tax', 'expense_id', 'tax_id',
         compute='_compute_from_product_id_company_id', store=True, readonly=False,
-        domain="[('company_id', '=', company_id), ('type_tax_use', '=', 'purchase')]", string='Taxes')
+        domain="[('company_id', '=', company_id), ('type_tax_use', '=', 'purchase'), ('price_include', '=', True)]", string='Taxes',
```

Comment:
Would you mind adapting the test (like how it was done [here] for master) to reflect price_include = True? :innocent: 

[here]: https://github.com/odoo/odoo/pull/81904/commits/5ac63e87e2c60a4f9fdc57285ed1a661eca47dfc#diff-799ab636d9774ba187a04433d7f836b860ce15a20c5e6f7dc414eebdb2bc670cR66-R72

Reply:
Hi Kevin, @kba-odoo, I'm not sure about the change at [L352](https://github.com/odoo/odoo/pull/81904/commits/5ac63e87e2c60a4f9fdc57285ed1a661eca47dfc#diff-799ab636d9774ba187a04433d7f836b860ce15a20c5e6f7dc414eebdb2bc670cL349-L352). Because it's related to another [fix](https://github.com/odoo-dev/odoo/commit/16459957d23f4bdfddfe96d72ef96cbdc6fec539#diff-799ab636d9774ba187a04433d7f836b860ce15a20c5e6f7dc414eebdb2bc670cR349-R352) few months ago. Could you have a check if this change should be included in the test?

Reply:
The changes you linked shouldn't be done in 15.0 indeed :-)
I think you can just do like [here] (write price_included = True on the taxes created) and it should be good :+1: 

Thanks!

[here]: https://github.com/odoo/odoo/pull/81904/commits/5ac63e87e2c60a4f9fdc57285ed1a661eca47dfc#diff-c97454a83c71e40b4c57d4bf947a2c7c34d1b549b3ee96f6fb4818590295ea24R55-R57

Reply:
Thank you for the hint! I also changed [L31](https://github.com/odoo/odoo/pull/81904/commits/5ac63e87e2c60a4f9fdc57285ed1a661eca47dfc#diff-272f0eb7a62e58834824418d79c87fc706bd2a7466f87964c25b3a1ff478060cL29-L31) in test_expenses_mail_import accordingly. Otherwise, it won't pass the test :)

Reply:
Hi Kevin, @kba-odoo, could you have a check on the new changes? Thanks!